src/controllers/userController.js

```diff
@@ -1,28 +1,39 @@
 import { PrismaClient } from '@prisma/client'
+import validateEmail from '../validation/EmailVailidation.js'
+import validateNumber from '../validation/PhoneValidation.js'
 
 const prisma = new PrismaClient()
 
 export const createUser = async function (req, res) {
   let data = req.body
   console.log(data)
-
-  const newUser = await prisma.user.create({
-    data
-  })
-
-  // console.log(newUser)
-  res.status(201).send({ status: true, newUser })
+  if (!validateEmail(data.email)) {
+    res.status(400).send({ status: false, message: 'Enter valid email' })
+  } else if (!validateNumber(data.phone)) {
+    res.status(400).send({ status: false, message: 'Enter valid phone number' })
+  } else {
+    const newUser = await prisma.user.create({
```

Comment:
Can we validate user presence before making POST call? like existence of same email in database. Prisma will throw  error based on Schema Restrictions